packages/input-amount/src/parsers.js

```diff
@@ -110,6 +110,10 @@ function parseHeuristic(value) {
  * @param {object} options Locale Options
  */
 export function parseAmount(value, options) {
+  const containsNumbers = value.match(/\d/g);
```

Comment:
I think this regex is readable enough that we don't need to store it in a variable with a descriptive name.

Reply:
I considered that indeed, but the intentions (stripping non numbers) are just a bit more clear like this.